tests/index.html

```diff
@@ -1 +1,103 @@
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta content='text/html;charset=utf-8' http-equiv='Content-Type'>
+    <title>RDF-star Test Suite</title>
+    <link rel='alternate' type="text/turtle" href='manifest.ttl'>
+    <link rel='alternate' type="application/ld+json" href='manifest.jsonld'>
+    <link href='https://www.w3.org/StyleSheets/TR/base' rel='stylesheet'>
+  </head>
+  <body>
+    <p>
+      <a href='http://www.w3.org/'>
+        <img alt='W3C' height='48' src='http://www.w3.org/Icons/w3c_home' width='72'>
+      </a>
+    </p>
+    <h1>RDF-star Test Suite</h1>
+    <p>This manifest loads additional manifests for specific behavior tests for
+      <a href="https://w3c.github.io/rdf-star">RDF-star</a></p>
+
+    <p>This is an HTML version of a test manifest.
+      Alternate versions of the manifest may be found at
+      <a href="manifest.jsonld">manifest.ttl</a>, and
+      <a href="manifest.jsonld">manifest.jsonld</a>.</p>
+
+    <p>The RDF-star Test Suite is a set of tests that can
+    be used to verify Processor conformance.</p>
+
+    <h2>General instructions for running the RDF-star Test suites</h2>
+
+    <p>Tests are run broadly the same as those for relavant
+      test suites for Turtle, SPARQL, and RDF 1.1 Entailment.</p>
+
+    <h1>Running tests</h1>
+
+    <p>The top-level <a href="manifest.ttl">manifest</a>
+      references the specific test manifests,
+      which in turn reference each test associated with
+      a particular behavior. This includes the following test manifests:</p>
+
+    <ul>
+      <li><a href="nt/syntax/">Notation-3 Syntax tests</a>,</li>
+      <li><a href="semantics/">Semantics tests</a>,</li>
+      <li><a href="sparql/syntax/">SPARQL Syntax tests</a>,</li>
+      <li><a href="sparql/eval/">SPARQL Evaluation tests</a>,</li>
+      <li><a href="turtle/syntax/">Turtle Syntax tests</a>,</li>
+      <li><a href="turtle/eval/">Turtle Evaluation tests</a>,</li>
+    </ul>
+
+    <p>Implementations create their own infrastructure for running the test suite.
+      In particular, the following should be considered:</p>
+
+    <ul>
+      <li>See <a href="https://w3c.github.io/rdf-tests/ntriples/">general guidance for running N-Triples tests</a>.</li>
+      <li>See <a href="https://w3c.github.io/rdf-tests/sparql11/">general guidance for running SPARQL 1.1 tests</a>.</li>
+      <li>See <a href="https://w3c.github.io/rdf-tests/turtle/">general guidance for running Turtle tests</a>.</li>
+      <li>See <a href="semantics/README">general guidance for running Semantics tests</a>.</li>
+    </ul>
+
+    <p>See the <a href="../reports/">implementation</a> report for guidance on reporting test
+      results using the EARL format.</p>
+
+    <h1>Contributing Tests</h1>
+
+    <p>If you would like to contribute a new test or a fix to an existing test,
+      please follow these steps:</p>
```

Comment:
Is there a reason not to suggest using github's largely automatic web interface, as opposed to what follows which appears to mandate using the manual clone/edit/PR process?

Reply:
If there's a simpler way, please recommend. I'm not familiar with a simple way to make all the changes typically needed to do such a change. There is a way to add or edit individual files, perhaps these could be coordinated into a single commit/PR. If there's a description of a purely web-based workflow, please let me know, or suggest an edit.

The intention of this is to lay out the steps typically necessary to submit such a change, not to mandate that as being the only way to make such a change.

Reply:
Yes, it's possible to change multiple files before submitting a PR, all through the web interface (which is how I operate).

Editing the first file automatically (and all but invisibly) triggers creation of a branch/clone. The web interface then suggests creating a PR immediately, but this is not required. Without closing that tab, I control-click on the automatically created branch name, and open it in a new tab (so I can most easily later use the auto-PR tools in the original tab). I again control-click on the other document(s) I want to change for this PR, opening each in a new tab. I make all the other changes connected to the first, and write them directly to the auto-created branch, closing each tab as the edits on that file are done. When all changes are done, I reload the tab that first prompted me to create a PR -- and that PR will now include the full DIFF between this branch and the original.

I can do this again, to create branch-2, etc., to build another PR with multiple files being changed, based on the original branch -- so, without the changes in the PR above.

The above isn't written with each and every step, because it's from memory, not from going through the motions, so I wouldn't include it in the README.  But I'd change "please follow these steps" to something along the lines of, "Use your preferred method to create and submit a Pull Request, if you know what would fix the problem you see, or open an Issue, if you see a problem, but don't know how to fix it."

Reply:
```suggestion
please use either the [GitHub web interface](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request), or use the following steps using a git client, or equivalent:</p>
```

Reply:
I think that change adequately describes both your workflow and the remote client mechanism. Feel free to suggest an improvement.

Reply:
Minor tweaks to grammar and a bit of wording 
```suggestion
    <p>If you would like to contribute a new test or a fix to an existing test,
please submit a Pull Request using either the <a 
href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request">GitHub web interface</a>, 
the steps below with a git client, or your preferred equivalent.</p>